torch/_inductor/cudagraph_trees.py

```diff
@@ -1660,7 +1660,7 @@ def check_memory_pool(device, pool_id, live_storages_ptrs: List[StorageWeakRefWr
         lambda: f"These storage data ptrs are not allocated in pool {pool_id} but should be {unique_storages}",
     )
 
-    if allocated_not_in_live_storages != 0:
+    if len(allocated_not_in_live_storages) != 0:
```

Comment:
thanks - at this point, we're guaranteed to have error'd haha, why this hasn't been an issue (but still good to fix).